pkg/kubelet/container/runtime.go

```diff
@@ -403,8 +403,6 @@ type Mount struct {
 
 // PortMapping contains information about the port mapping.
 type PortMapping struct {
-	// Name of the port mapping
-	Name string
```

Comment:
simplifying code review: this field used to be needed for the implementaiton of port mapping inside dockershim. Not anymore. CNI doesn't care about the name of the mapping.